core/modules/config.js

```diff
@@ -38,4 +38,18 @@ exports.htmlBlockElements = "address,article,aside,audio,blockquote,canvas,dd,de
 
 exports.htmlUnsafeElements = "script".split(",");
 
+// Character code lookups used in charcode operator and macro
+exports.lookup = {
```

Comment:
Would recommend use of a more explicit name that makes it clear what this is for, for example charcodeLookup rather than just lookup.

Reply:
good idea